### PR TITLE
adsinteractive: added usersync endpoint

### PR DESCRIPTION
**Pull request by balintvargha**
Dear Prebid Team!

We added  a usersync endpoint to our existing adapter, please review and let us know if there's anything further we must set in order for it to work.

### CODE REVIEW

File: static/bidder-info/adsinteractive.yaml [hunk 1]
```diff
@@ -8,4 +8,9 @@ capabilities:
       - banner
   site:
     mediaTypes:
-      - banner
\ No newline at end of file
+      - banner
+userSync:
+  redirect:
+    url: "http://pb.adsinteractive.com/getuid?{{.RedirectURL}}"
+    userMacro: "$AUID"
```
Comment (gargcreation1992) [inline static/bidder-info/adsinteractive.yaml:15]: This redirect URL doesn't seem to be redirecting to redirect URL and setting UID macro. Could you please check?
Here is the URL that I tried - 
```
http://pb.adsinteractive.com/getuid?https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dadsinteractive%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24AUID
```

Comment (gargcreation1992) [reply static/bidder-info/adsinteractive.yaml:15]: @balintvargha The URL is still not redirecting and adding the UID macro.

Comment (onkarvhanumante) [reply static/bidder-info/adsinteractive.yaml:15]: @balintvargha `$AUID` macro is not replaced with uuid

Screenshot 2023-04-11 at 10 18 08 PM

Comment (balintvargha) [reply static/bidder-info/adsinteractive.yaml:15]: $AUID macro is now being replaced by uuid, please check now.

Comment (onkarvhanumante) [reply static/bidder-info/adsinteractive.yaml:15]: endpoint is working. Thanks @balintvargha for following up on this

**Review by onkarvhanumante — APPROVED** [review]

File: static/bidder-info/adsinteractive.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/adsinteractive.yaml:15]: look good now. thanks for being patient on this one!

**Review by gargcreation1992 — APPROVED** [review]

File: static/bidder-info/adsinteractive.yaml (hunk 1 above)
Comment (balintvargha) [reply static/bidder-info/adsinteractive.yaml:15]: Thank you @onkarvhanumante and @gargcreation1992 for reviewing, also for your patience as well

**Pull request merged**